truflation/truflation
# feat: list exported csv/json files in an optional manifest file

Test steps:

```
docker build -t truflation-data .
cp examples/csv_example/.env_template .env
docker run -it -v .:/home/user truflation-data bash
```
inside docker container:
```
cd /home/user
pipeline_run_direct.py examples/csv_example/my_pipeline_details.py
```
observe that the output/manifest file is created

changing to `CONNECTOR=json:json` also works
